packages/psss-server/src/connections/EntityConnection.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { PSSS_ENTITY, PSSS_HIERARCHY } from '../constants';
+import { ApiConnection } from './ApiConnection';
+
+const { ENTITY_URL = 'http://localhost:8050/v1' } = process.env;
+
+type EntityObject = {
+  id: string;
+  code: string;
+  name: string;
+};
+
+interface EntityFilter {
+  type?: string | string[];
+}
```

Comment:
@rohan-bes We could use some TS magic here by importing the fuller types defined in `entity-server`, but then again we'd make `psss-server` dependent on `entity-server`, if only for `dev` purposes.

Keen to hear about ways to share types between orchestration servers and microservices! For the time being, I thought that defining what we need should be enough here.

Reply:
Hmmm it's a good question... I think for now it's fine for us to keep them decoupled... Perhaps in the future it might be nice for each micro-service to define some exposed interface types that clients can use to correctly format requests... I think we still need some time to see how things evolve before trying to tightly couple everything 😄

Reply:
I agree!